lib/order.rb

```diff
@@ -0,0 +1,135 @@
+require 'csv'
+require 'awesome_print'
+require_relative 'customer.rb'
+
+class Order
+
+attr_reader :id
+
+# The fulfillment_status = :pending sets a default value for fulfillment_status
+# Note: @products is a single hash that contains key: food_name and value: price
+# Example: @products = {"banana" => 2.25, "cookie" => 1.50}
+# The constructor's customer parameter is a Customer object NOT customer id
+def initialize(id, products, customer, fulfillment_status = :pending)
+  @id = id
+  @products = products
+  @customer = customer
+
+  if fulfillment_status == :pending || fulfillment_status == :paid || fulfillment_status == :processing || fulfillment_status == :shipped || fulfillment_status == :complete
```

Comment:
This is pretty good -- is there another way we can check if `fulfillment_status` is one of a collection of things in a different form?